modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -34,36 +34,73 @@ Establishing the right performance expectations refers to the fact that the real
 == Provisioning a worker with real-time capabilities
 
 . Install Performance Addon Operator to the cluster.
+
 . Optional: Add a node to the {product-title} cluster.
 See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_for_real_time/8/html-single/tuning_guide/index#Setting_BIOS_parameters[Setting BIOS parameters].
-. Optional: Create a new machine config pool for real-time nodes.
-. Add the node to the proper machine config pool, using node role labels.
-+
-You must decide which nodes will be configured with real-time workloads. It could be all of the nodes in the cluster or a subset of the nodes. The Performance Addon Operator expects all of the nodes are part of a dedicated machine config pool. If you use all of the nodes, you just point the Performance Addon Operator to the worker node role label. If you use a subset, you must group the nodes into a new machine config pool.
 
-. Create the `PerformanceProfile` with the proper set of housekeeping cores and `realTimeKernel: enabled: true`.
+. Add a label `worker-rt` to the worker nodes that require the real-time capability using the `oc` command.
 
-. Specify a node selector in the `PerformanceProfile`, as shown here:
+. Create a new machine config pool for real-time nodes:
 +
 [source,yaml]
 ----
-apiVersion: performance.openshift.io/v2
-kind: PerformanceProfile
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
 metadata:
- name: example-performanceprofile
+  name: worker-rt
+  labels:
+    machineconfiguration.openshift.io/role: worker-rt
 spec:
-...
-  realTimeKernel:
-    enabled: true
+  machineConfigSelector:
+    matchExpressions:
+      - {
+           key: machineconfiguration.openshift.io/role,
+           operator: In,
+           values: [worker, worker-rt],
+        }
+  paused: false
   nodeSelector:
-    node-role.kubernetes.io/worker-rt: ""
+    matchLabels:
+      node-role.kubernetes.io/worker-rt: ""
 ----
+Note that a machine config pool worker-rt is created for group of nodes that have the label `worker-rt`.
+
+. Add the node to the proper machine config pool, using node role labels.
++
+You must decide which nodes will be configured with real-time workloads. It could be all of the nodes in the cluster or a subset of the nodes. The Performance Addon Operator expects all of the nodes are part of a dedicated machine config pool. If you use all of the nodes, you just point the Performance Addon Operator to the worker node role label. If you use a subset, you must group the nodes into a new machine config pool.
 
+. Create the `PerformanceProfile` with the proper set of housekeeping cores and `realTimeKernel: enabled: true`.
```

Comment:
Should I know what is proper in this case?

Reply:
> and `realTimeKernel: enabled: true`.

This should be expanded, e.g. "and the`realTimeKernel` resource's `enabled` property set to `true`."

Reply:
Should I know what is proper in this case>> I am not sure as this is existing content..